### PR TITLE
Schema work

### PR DESCRIPTION
**Pull request by jeffweeksio**
This includes fixes for issues 1 and 3.  Still need to add area/artist/label boosts and recording/release-group similarity.  FreeDB is included in this but, is being deprecated (per Rob) so not in sir.  

This is my 1st ever pull request so let me have it!

### CODE REVIEW

File: cdstub/conf/common [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/jeff/mbsssss/common
```
Comment (mineo) [inline cdstub/conf/common]: All the symbolic links that point to `/home/jeff` will not work on any computer that doesn't have the repository in there. You'll need to recreate these as relative paths with `ln -s ../../common .`

**Comment by mineo** [comment]
The `write.lock` files and ` recording/data/index/segments_1` should not be part of the git repository, they're managed by solr at runtime.

File: index/.project [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mineo) [inline index/.project]: Does this file need to be checked in for something to work? It's the only file in the `index` folder and only seems to cnotain eclipse settings.

**Comment by jeffweeksio** [comment]
I fixed my noobie mistakes.  Thanks for catching them!

**Comment by jeffweeksio** [comment]
I figured out how to add issues to my branch.  Some of my issues duplicate yours, but I seperated them out into more specific issues.

File: README.md [hunk 3]
```diff
@@ -9,15 +9,29 @@ Note that this repository uses the
...
-[the MusicBrainz QueryResponseWriter](https://github.com/mineo/mb-solrquerywriter)
-into the `lib` folder, which is configured as the Solr servers **sharedLib**
+[the MusicBrainz QueryResponseWriter](https://github.com/jeffweeksio/mb-solrquerywriter)
+and place the resulting file called solrwriter-0.0.1-SNAPSHOT-jar-with-dependencies.jar into the `mbsssss/lib` folder, which is configured as the Solr server's **sharedLib**
 folder.
 
-Then download Solr from its
+## Adding the AliasSimilarity and ReleaseSimilarity Classes
+
+If for some reason you're not already there, you will need to navigate to the `mbsssss/lib` folder in a terminal and build another JAR
```
Comment (mineo) [inline README.md]: @jeffweeksio That folder doesn't seem to include the java files, do you still have them and could commit them?

Comment (jeffweeksio) [inline README.md]: For some reason, I'm thinking those might not be needed any more and I forgot to change the Readme to reflect that.  I need to get onto the server to know for sure.  I'm awaiting the addition of my new public key to that server for access.

Comment (mineo) [inline README.md]: They're at least still referenced in the `text_mult_alias_similarity` field type.

Comment (jeffweeksio) [inline README.md]: It looks like I added them to the mb-solrquerywriter so a 2nd JAR didn't need to be built. mb-solrquerywriter/src/main/java/org/musicbrainz/search/analysis/

**Pull request merged**